spec/factories/course_forum_topics.rb

```diff
@@ -1,12 +1,20 @@
 # frozen_string_literal: true
 FactoryGirl.define do
   factory :forum_topic, class: Course::Forum::Topic.name do
-    forum
+    transient do
+      course { build(:course) }
```

Comment:
Since `:forum_topic` now `acts_as :topic`, do we need to add a parent to the factory? 

So it becomes:
```ruby
FactoryGirl.define do
  factory :forum_topic, class: Course::Forum::Topic.name, parent: course_discussion_topic do 
    ...
  end
end
```